share-link lib: jQuery to vanilla JS [ED-9706]

nuritsha opened the pull request:
## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* [tweak] share-link lib: replace the use of jQuery with vanilla JS

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

matipojo committed:
Internal: AI - Move remote config loading to frontend [ED-15461] (#28229)

ManorHazaz committed:
Internal: Add atomic widgets style command [EDS-257] (#28112)

KingYes committed:
Fix: Element Caching prevents Icon List Items from loading Dynamic Tags (#28137) [ED-15421] (#28232)

DennisNerush committed:
Fix: Modal doesn't adjust height to content in AI Text [ED-15478] (#28249)

arielk committed:
Internal: Implementing new animated cover image in WP Repo [ED-15168] (#28251)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>
Co-authored-by: Yakir Sitbon <kingyes1@gmail.com>
Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>
Co-authored-by: Rami Yushuvaev <92088692+rami-elementor@users.noreply.github.com>

nuritsha committed:
share-link lib: jQuery to vanilla JS [ED-9706]

rami-elementor committed:
Tweak: Add heading and descriptions to admin settings pages [ED-15322] (#28183)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

gabriel-glo committed:
Tweak: Floating Buttons - accessibility improvements (#28037)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* New Accessible Name field for Floating Buttons widget

## Description
An explanation of what is done in this PR

* Add `aria-role` attribute to main wrapper of the widget
* Add `aria-label` attribute to main wrapper of the widget
* Add a control to widget setting for managing `aria-label` attribute

## Test instructions
This PR can be tested by following these steps:

* Add and Floating Button widget and change the default Accessible Name
setting

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Maggie Serino <maggie.serino@10up.com>
Co-authored-by: Maggie Serino <7341759+mserino@users.noreply.github.com>

ramiy commented on assets/lib/share-link/share-link.js:
```diff
@@ -1,10 +1,10 @@
-		var $element,
-			settings = {};
+( function() {
+	window.ShareLink = function( eElement, userSettings ) {
```
Is `eElement` used in the code?

nuritsha replied:
hmm, I don't see it, but is doesn't make sense, there should be.

ramiy commented on assets/lib/share-link/share-link.js:
```diff
@@ -1,10 +1,10 @@
-			settings = {};
+( function() {
+	window.ShareLink = function( eElement, userSettings ) {
+		var element,
```
Please consider replacing `var` with `let`/`const`.

ramiy commented on assets/lib/share-link/share-link.js:
```diff
@@ -1,10 +1,10 @@
+( function() {
+	window.ShareLink = function( eElement, userSettings ) {
+		var element,
+			setting = {};
```
Lastly, let's stop using comma separate declaration.

nuritsha committed:
ES6

nuritsha committed:
typo

rami-elementor committed:
Internal: Add widget style loading method [ED-15488] (#28253)

rami-elementor committed:
Fix: The 'Elementor settings' links inside the 'Theme Style' are broken [ED-15490] (#28258)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

arielk committed:
Internal: Implementing new animated cover image in WP Repo [ED-15492] (#28262)

StyleShit committed:
Internal: Atomic widgets props schema default [EDS-282] (#28201)

nicoladj77 committed:
Internal: Floating Buttons - Inline editing variation 1 (#28235)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Add inline editing support for variation 1

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>
Co-authored-by: Maggie Serino <7341759+mserino@users.noreply.github.com>

Nevoss committed:
Internal: Replace actions with reusable actions [EDS-292] (#28199)

netanelavr committed:
Fix: Ensure the wp request action suffix is sent [ED-15505] (#28269)

nicoladj77 committed:
Fix: Floating Buttons - Max Width on mobile [ED-15502] (#28276)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix full width mobile bar

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

rami-elementor commented on assets/lib/share-link/share-link.js:
```diff
@@ -29,26 +27,26 @@
-			var link = ShareLink.getNetworkLink( networkName, settings ),
+			const link = ShareLink.getNetworkLink( networkName, settings ),
 				isPlainLink = /^https?:\/\//.test( link ),
 				windowName = isPlainLink ? '' : '_self';
```
Same here (lines 40-42), stop using comma separate declaration.

StyleShit committed:
Internal: Throw exception when binding a non-existing prop [EDS-297] (#28268)

nuritsha committed:
after code review

rami-elementor committed:
Tweak: Add "Performance" and “Integrations“ links to Elementor Finder [ED-15511] (#28284)

nuritsha committed:
Merge branch 'main' into tweak/ED-9706

hein-obox commented:
@nuritsha 

Please can you create a PR for the `main` branch, and not for the `master` branch.

hein-obox commented:
@nuritsha 

Please can you create a PR for the `main` branch, and not for the `master` branch.

nuritsha commented:
@hein-obox please note that this PR is on my fork, this one is on the elementor repo: https://github.com/elementor/elementor/pull/28285

hein-obox commented:
@nuritsha Maybe can you close this PR, and use the other PR?

louiswol94 committed:
Internal: Fix home screen import (#28289)

TzviRabinovitch committed:
Internal: Changelog v3.23.3 (#28290)

ManorHazaz committed:
Fix: Styles Command - history transaction [EDS-331] (#28287)

yotamselementor committed:
Internal: Remove old and unused GitHub actions (#28288)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

rami-elementor committed:
Tweak: Conditionally load widget styles only if the widgets are used [ED-15384] (#28128)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>
Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>

hein-obox committed:
Tweak: Load Lightbox styles only when it is in use [ED-15042] (#28271)

ManorHazaz committed:
Internal: Atomic Heading - Add styles classes [EDS-335] (#28300)

Co-authored-by: StyleShit <evyatar.daud@gmail.com>

rami-elementor committed:
Tweak: Promoted "Editor Top Bar" experiment to Stable status [ED-15413] (#28305)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

baghdasarovelementor committed:
Fix: Change Inspector for Shop Page Edit [ED-10199] (#28309)

ronros-elementor committed:
Internal: Load editor style assets [EDS-315]  (#28260)

dependabot[bot] committed:
Internal: Bump the elementor-packages group across 1 directory with 14 updates (#28307)

nuritsha committed:
after code review

ronenelementor committed:
Internal: Create The Marketing Checklist BE Infra [ED-15465] (#28274)

Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>

yotamselementor committed:
Internal: Stabilize Clone multiple tab items test step (#28316)

yotamselementor committed:
Internal: Use API for uninstalling plugins in Plugin Tester Generator (#28317)

nicoladj77 committed:
Internal: Floating Buttons - Prevent CPT from showing up in query controls [ED-15491] (#28265)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Ensure that Floating Buttons do not show up in query controls

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>
Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>

StyleShit committed:
Internal: Improve atomic widget tests (#28321)

DennisNerush committed:
Internal: Direct link to Connect > Build with AI flow [ED-15500] (#28314)

https://www.loom.com/share/6f97acf8f6f74a7294457ee7610993a7?sid=e0792a3d-6f09-4c3e-a4c1-791e82c5feb1

---------

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

nicoladj77 committed:
Internal: Floating Elements - Refactor the admin screen to allow choosing the floating element type (#28264)

DennisNerush committed:
Internal: Add Taxonomy to AI context as it's no longer an experiment [ED-15530]  (#28329)

rami-elementor committed:
Tweak: Conditionally load widgets styles only if the widgets are used [ED-15384] (#28328)

Relates to https://github.com/elementor/elementor/pull/28128

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

DennisNerush committed:
Tweak: Added ability to opt-out AI features per user (#22796, #22853) [ED-15000]  (#28303)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

rami-elementor committed:
Tweak: Activated "Nested Elements" experiment to new sites [ED-15532] (#28336)

hein-obox committed:
Tweak: Load dialog styles only when it is in use [ED-15214] (#28301)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Hein van Vlastuin <heinvanvlastuin@gmail.com>
Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

nuritsha committed:
after code review

nuritsha committed:
Merge branch 'main' into tweak/ED-9706

nuritsha committed:
after review

nuritsha committed:
use arrow functions

ronenelementor committed:
Internal: Class fix for better maintainability (#28348)

naorsabag committed:
Internal: Added WooCommerce filtering support for text based containers [ED-15540] (#28351)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Add filtering of WooCommerce plugin to the AI text based containers

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

rodolphebertozzo committed:
Tweak: Refactoring for use margin-block-start instead of margin-top on… (#24646)

...caption spacing

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>
Co-authored-by: Ariel Klikstein <ariel@elementor.com>

rami-elementor committed:
Fix: Displays "Custom Positioning" navigator-indicator when "width" is set [ED-15541] (#28352)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

hein-obox committed:
Fix: Fix Dialog.js console error [ED-15544] (#28358)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Co-authored-by: Hein van Vlastuin <heinvanvlastuin@gmail.com>
Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

mserino committed:
Fix: Floating Buttons - responsive controls (#28326)

nicoladj77 committed:
Tweak: Link In Bio - Remove experiment [ED-15509] (#28347)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Merge Link In Bio into main

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>

dependabot[bot] committed:
Internal: Bump the elementor-packages group across 1 directory with 2 updates (#28366)

hein-obox committed:
Fix: Check if has elementorFrontend object [ED-15544] (#28369)

netanelavr committed:
Tweak: Preview result in AI CSS feature [ED-14993] (#28360)

mserino committed:
New: Floating Bars - Variation 3 (#28283)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

mserino committed:
New: Floating Bars - Variation 2 - ticker (#28261)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>

elchugreeva committed:
Internal: Create the Checklist "Step" React component [ED-15466] (#28310)

Co-authored-by: David Seveloff <davids@elementor.com>
Co-authored-by: Ronen Boxer <ronenb@elementor.com>
Co-authored-by: Ronen Boxer <145427686+ronenelementor@users.noreply.github.com>
Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

mugurelLupu committed:
Tweak: Swiper loads css only when needed [ED-15015] (#28299)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Hein van Vlastuin <heinvanvlastuin@gmail.com>
Co-authored-by: Hein van Vlastuin <94352322+hein-obox@users.noreply.github.com>
Co-authored-by: Rami Yushuvaev <92088692+rami-elementor@users.noreply.github.com>

nicoladj77 committed:
Fix: Floating Elements - Move Bar based on position and fix library (#28345)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Move Floating Bar based on position

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Maggie Serino <7341759+mserino@users.noreply.github.com>
Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>

nicoladj77 committed:
Fix: Floating Elements - Hide floating bars (#28371)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Hide floating bars

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

Co-authored-by: Louis Wolmarans <louiswol94@gmail.com>
Co-authored-by: Maggie Serino <7341759+mserino@users.noreply.github.com>

IshayMaya committed:
Internal: Add atomic image [EDS-321] (#28322)

nicoladj77 committed:
Fix: Floatign Buttons - when library is empty, redirect to admin on close (#28377)

StyleShit committed:
Internal: Add `@elementor/wp-media` package [EDS-356] (#28357)

mserino committed:
Tweak: Floating Bars - tweaks and fixes (#28374)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

rami-elementor committed:
Internal: Replace style loading function for wp-admin-bar [ED-15555] (#28382)

hein-obox committed:
Internal: Move the conditional loading of Swiper styles [ED-15548] (#28373)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

---------

Co-authored-by: Hein van Vlastuin <heinvanvlastuin@gmail.com>
Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

ManorHazaz committed:
Internal: Add atomic attachment control [EDS-320] (#28361)

TzviRabinovitch committed:
Internal: Bump to 3.25 (#28385)

romanserkelem committed:
New: Added Cleanup image edit tool [ED-13777] (#28381)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:


## Description
This PR adds a cleanup button to the 'Edit with AI' window. The cleanup
screen allows users to select and mask the object they want to remove.
After removal, a new image with the object removed is generated. The
cancel and regenerate options appear after the cleanup is applied.


## Quality assurance

- [X] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

---------

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

rami-elementor committed:
Tweak: Conditionally load widgets styles only if the widgets are used [ED-15384] (#28387)

Related to:
https://github.com/elementor/elementor/pull/28128
https://github.com/elementor/elementor/pull/28328

---------

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

nuritsha committed:
Merge branch 'main' into tweak/ED-9706
